pkg/nimble/autoconn/include/nimble_autoconn_params.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_nimble_autoconn
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for the nimble_autoconn module
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef NIMBLE_AUTOCONN_PARAMS_H
+#define NIMBLE_AUTOCONN_PARAMS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Default parameters used for the nimble_netif_autoconn module
```

Comment:
If this are configurations then add `@ingroup     config`

Reply:
Partly yes and no. For now I would like to leave them out of the config module - I need to figure out first how to best present the information there... So I will fix this once the module stabilizes at some point.

Reply:
> I need to figure out first how to best present the information there... So I will fix this once the module stabilizes at some point.

So If i understand, some of these should not be configurable? or maybe you are still figuring out the right range of possible values for this to work right?